src/components/CommanderOption.tsx

```diff
@@ -11,15 +11,16 @@ type Props = {
 const CommanderOption: React.FC<Props> = ({ card, isSelected, onSelect }) => {
     const className = clsx(
         'c-commander-option', {
-            'c-commander-option--selected': isSelected
-        },
-    );
+        'c-commander-option--selected': isSelected
+    });
 
     return (
         <button type="button" className={className} onClick={onSelect}>
-            <img src={card.frontFace.artImageUri} alt={card.frontFace.name} width={250} height={183} />
-            <span className="c-commander-option__name">{card.frontFace.name}</span>
-            <span className="c-commander-option__artist">art by {card.frontFace.artist}</span>
+            <img src={card.frontFace.fullImageUri} alt={card.frontFace.name} />
+            <div>
```

Comment:
NIT: You can't have a div inside a button, would you be able to replace it with a span or remove it?